components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```diff
@@ -207,7 +207,20 @@ def search_and_schedule_new_tasks(
 
         input_type = input_config.type
         if input_type == InputType.FS.value:
-            _process_fs_input_paths(input_config, paths_to_compress_buffer)
+            try:
+                _process_fs_input_paths(input_config, paths_to_compress_buffer)
+            except Exception as err:
+                logger.exception("Failed to process one or more file input paths.")
```

Comment:
nit: maybe we should do:
1. store `Failed to process input: {str(err)}` as an error_msg variable
2. use it both in `logger.exception` and for `update_compression_job_metadata`?